src/tasks/gitea/gitea.ts

```diff
@@ -0,0 +1,34 @@
+import { RepositoryApi, CreateRepoOption } from '@redkubes/gitea-client-node'
+
+import { cleanEnv, GITEA_USER, GITEA_PASSWORD, GITEA_URL, GITEA_REPO } from '../../validators'
+
+const env = cleanEnv({
+  GITEA_USER,
+  GITEA_PASSWORD,
+  GITEA_URL,
+  GITEA_REPO,
```

Comment:
I thought we would hardcode the values repo and user. It makes no sense to offer configuration for this

Reply:
These values are hardcoded in the job: https://github.com/redkubes/otomi-core/blob/gitea/values/jobs/gitea.gotmpl#L11-L13

I think it makes more sense to hardcode it there, then in a task.

Reply:
ok, lets move on with this